test/e2e/cloud-provider-azure.go

```diff
@@ -61,6 +61,10 @@ func InstallCalicoAndCloudProviderAzureHelmChart(ctx context.Context, input clus
 	clusterProxy := input.ClusterProxy.GetWorkloadCluster(ctx, input.ConfigCluster.Namespace, input.ConfigCluster.ClusterName)
 	InstallHelmChart(ctx, clusterProxy, defaultNamespace, cloudProviderAzureHelmRepoURL, cloudProviderAzureChartName, cloudProviderAzureHelmReleaseName, options)
 
+	if input.ConfigCluster.Flavor == "flatcar" {
```

Comment:
This is not what I suggested in https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/3387#issuecomment-1495847242, which is why tests are still failing.

Reply:
Sorry, I might be missing something obvious. Can you please explain what is that I am missing? 
Aren't these [the lines of code](https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/8b9b3bf5eceabb80f2752e303b60e89ad919b5d1#diff-1131bb4107662928d2a952d842948b3d7ae633a00339807f164ac7446ad807c5R61-R63) we are trying to add to `test/e2e/cloud-provider-azure.go` ?

Reply:
This code should be executed before CCM is installed, since it configures values for CCM chart, not after.

Reply:
🤦🏼‍♂️ Of course...